### PR TITLE
Error Page Added

### PR DESCRIPTION
**Pull request by PatilHarshh**
## Related Issue

- Error Page Added

Closes: #269 
## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| original screenshot | updated screenshot |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by PatilHarshh** [comment]
@vinzvinci Please Merge it ...
Learnt some thing new and worked first time in Next js

File: app/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+          <div className="font-dark mb-8 mt-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
+
+          <div className="font-dark mb-8 mt-8 text-2xl  text-gray-400">
+            It seems like you wandered off too far! You must return back to our guild!
+          </div>
+
+          <a
+            href="/"
+            className="duration-400 inline rounded-lg border border-transparent bg-blue-600 px-6 py-4 text-base font-medium leading-6 text-white shadow-2xl transition-all hover:bg-red-700 focus:outline-none active:bg-red-600"
+
```
Comment (krishstwt) [inline app/not-found.tsx:22]: This empty line is useless
```suggestion
```

**Comment by PatilHarshh** [comment]
done @krishstwt

File: app/not-found.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Image from "next/image";
+
+export default function NotFound() {
+  return (
+    <div className="flex h-screen w-screen items-center bg-gray-50">
+      <div className="container flex flex-col items-center justify-center px-5 text-gray-700 md:flex-row">
+        <div className="mx-8 w-full text-center lg:w-1/2">
+          <div className="font-dark mb-8  text-7xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Oops!</div>
+          <div className="flex justify-center">
+            <Image src="/Frame.png" layout="responsive" width={300} height={300} alt="404 Error" />
```
Comment (Anmol-Baranwal) [inline app/not-found.tsx:10]: `Layout` is deprecated. 

Use `fill` and refer to the documentation here:
https://nextjs.org/docs/pages/api-reference/components/image#fill

File: app/not-found.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          </div>
+          <div className="font-dark mb-8 mt-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
```
Comment (Anmol-Baranwal) [inline app/not-found.tsx:12]: - `mb-8 mt-8` can be replaced by `my-8`
```suggestion
          <div className="font-dark my-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
```
- what is the use of `font-dark`

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
1. Please make it responsive since it is overflowing at a certain screen size.

> Image 1
image

> Image 2
image

2. Please change the hover color from red to something slightly different from the primary color. Maybe adjust the opacity so that a minor change is visible on hovering.

<hr>

> If you need help with the code, just let me know.

**Comment by PatilHarshh** [comment]
hey @Anmol-Baranwal I Recently make changes if still not responsive please provide me at which length its failing and button hovering also changed

File: app/not-found.tsx [hunk 4]
```diff
@@ -5,18 +5,17 @@ export default function NotFound() {
...
-            <Image src="/Frame.png" layout="responsive" width={300} height={300} alt="404 Error" />
+            <Image src="/Frame.png"  width={300} height={300} alt="404 Error" />
           </div>
-          <div className="font-dark mb-8 mt-8  text-4xl font-bold "style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
-          <div className="font-dark mb-8 mt-8 text-2xl  text-gray-400">
+          <div className=" my-8 text-4xl  font-bold"style={{ color: 'rgba(65, 26, 255, 1)' }}>Page not found</div>
+          <div className=" my-8 text-2xl text-gray-400">
             It seems like you wandered off too far! You must return back to our guild!
           </div>
           <a
```
Comment (Anmol-Baranwal) [inline app/not-found.tsx:16]: You can add `aria-label="Return to Home"` to assist screen readers.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me. Thanks for the changes.

**Comment by PatilHarshh** [comment]
@Anmol-Baranwal Merging?

**Comment by Anmol-Baranwal** [comment]
> @Anmol-Baranwal Merging?

Yeah, let vinzvinci review this.

**Review by vinzvinci — APPROVED** [review]
LGTM! @WebXDAO/gssoc-mentors add labels

**Review by krishstwt — APPROVED** [review]

**Comment by Anmol-Baranwal** [comment]
> LGTM! @WebXDAO/gssoc-mentors add level labels

Since this is an entirely new feature or component, I am assigning it a `level3`. Feel free to change it if you think otherwise.

**Pull request merged**